# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/sysvinit/sysvinit.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/Makefile.orig	Mon May  6 11:10:51 2002
+++ ./src/Makefile	Mon May  6 11:12:55 2002
@@ -14,12 +14,8 @@
 STATIC	=
 
 # For Debian we do not build all programs, otherwise we do.
-ifeq ($(DEBIAN),)
-PROGS	= init halt shutdown killall5 runlevel sulogin utmpdump \
+PROGS	= init halt shutdown killall5 runlevel sulogin \
 		last mesg wall
-else
-PROGS	= init halt shutdown killall5 runlevel sulogin last mesg
-endif
 
 BIN_OWNER	= root
 BIN_GROUP	= root
@@ -46,9 +42,6 @@
 mesg:		mesg.o
 		$(CC) $(LDFLAGS) -o $@ mesg.o
 
-utmpdump:	utmpdump.o
-		$(CC) $(LDFLAGS) -o $@ utmpdump.o
-
 runlevel:	runlevel.o
 		$(CC) $(LDFLAGS) -o $@ runlevel.o
 
@@ -88,9 +81,7 @@
 		$(INSTALL) -m 755 halt init killall5 sulogin \
 			runlevel shutdown $(ROOT)/sbin
 		# These are not installed by default
-ifeq ($(DEBIAN),)
-		$(INSTALL) -m 555 utmpdump wall $(ROOT)/usr/bin
-endif
+		$(INSTALL) -m 555 wall $(ROOT)/usr/bin
 		# $(INSTALL) -m 755 etc/initscript.sample $(ROOT)/etc
 		$(INSTALL) -m 755 mesg last $(ROOT)/usr/bin
 		cd $(ROOT)/sbin; ln -sf halt reboot; chown $(BIN_COMBO) reboot
@@ -101,9 +92,7 @@
 		$(INSTALL) -m 644 initreq.h $(ROOT)/usr/include
 		$(INSTALL) -m 644 ../man/*.8 $(ROOT)$(MANDIR)/man8
 		$(INSTALL) -m 644 ../man/*.5 $(ROOT)$(MANDIR)/man5
-ifeq ($(DEBIAN),)
 		$(INSTALL) -m 644 ../man/wall.1 $(ROOT)$(MANDIR)/man1
-endif
 		$(INSTALL) -m 644 ../man/last.1 ../man/lastb.1 ../man/mesg.1 \
 			$(ROOT)$(MANDIR)/man1
 		#
